docs/getting-started/mainnet_validator.md

```diff
@@ -0,0 +1,453 @@
+
+  ```bash
+  $ sed -i.bak -E 's#^(persistent_peers[[:space:]]+=[[:space:]]+).*$#\1"87c3adb7d8f649c51eebe0d3335d8f9e28c362f2@seed-0.crypto.org:26656,e1d7ff02b78044795371beb1cd5fb803f9389256@seed-1.crypto.org:26656,2c55809558a4e491e9995962e10c026eb9014655@seed-2.crypto.org:26656"#' ~/.chain-maind/config/config.toml
+  $ sed -i.bak -E 's#^(seeds[[:space:]]+=[[:space:]]+).*$#\1""#' ~/.chain-maind/config/config.toml
+  $ LATEST_HEIGHT=$(curl -s https://mainnet.crypto.org:26657/block | jq -r .result.block.header.height); \
+  BLOCK_HEIGHT=$((LATEST_HEIGHT - 1000)); \
+  TRUST_HASH=$(curl -s "https://mainnet.crypto.org:26657/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
+
+  $ sed -i.bak -E "s|^(enable[[:space:]]+=[[:space:]]+).*$|\1true| ; \
+  s|^(rpc_servers[[:space:]]+=[[:space:]]+).*$|\1\"https://mainnet.crypto.org:26657,https://mainnet.crypto.org:26657\"| ; \
+  s|^(trust_height[[:space:]]+=[[:space:]]+).*$|\1$BLOCK_HEIGHT| ; \
+  s|^(trust_hash[[:space:]]+=[[:space:]]+).*$|\1\"$TRUST_HASH\"| ; \
+  s|^(seeds[[:space:]]+=[[:space:]]+).*$|\1\"\"|" ~/.chain-maind/config/config.toml
+  ```
+
+    ::: tip NOTE
+
+  - For Mac environment, if `jq` is missing, you may install it by: `brew install jq`
+    :::
+
+**Note**: We suggest using `persistent_peers` instead of `seeds` to provide stable state-sync experience.
+
+## Step 3. Run everything
+
+### Step 3-1. Run everything
+
+Once the `chain-maind` has been configured, we are ready to start the node and sync the blockchain data:
+
+- Start `chain-maind`, e.g.:
+
+```bash
+  $ ./chain-maind start
+```
+**OR**
+- _(Optional for Linux)_ If you would like to have it running at the background, you can start `chain-maind` with `systemd` service, e.g.:
+
+```bash
+  $ git clone https://github.com/crypto-org-chain/chain-main.git && cd chain-main
+  $ ./networks/create-service.sh
+  $ sudo systemctl start chain-maind
+  # view log
+  $ journalctl -u chain-maind -f
+```
+
+:::details Example: /etc/systemd/system/chain-maind.service created by script
+
+```bash
+# /etc/systemd/system/chain-maind.service
+[Unit]
+Description=Chain-maind
+ConditionPathExists=/usr/local/bin/chain-maind
+After=network.target
+
+[Service]
+Type=simple
+User=ubuntu
+WorkingDirectory=/usr/local/bin
+ExecStart=/usr/local/bin/chain-maind start --home /home/ubuntu/.chain-maind
+Restart=on-failure
+RestartSec=10
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target
+```
+
+:::
+
+It should begin fetching blocks from the other peers. Please wait until it is synced before moving onto the next step.
+
+::: tip Remarks: 
+  - You can query the node syncing status by
+    ```bash
+    $ ./chain-maind status 2>&1 | jq '.SyncInfo.catching_up'
+    ```
+    If the above command returns `false`, It means that your node **is synced**; otherwise, it returns `true` and implies your node is still catching up.
```

Comment:
```suggestion
    If the above command returns `false`, it means that your node **is synced**; otherwise, it returns `true` and implies your node is still catching up.
```